design/system-snapshot.md

```diff
@@ -67,12 +71,16 @@ snapshot. The FATE op would do the following :
  1. get snapshot lock in ZK (prevents concurrent snapshot operations)
  1. ensure snapshot name is not in use
  1. pause changing props in ZK
- 1. pause non-snapshot fate ops (let them finish current step, but do not
-    execute next step).   Temporarily stop accepting new FATE ops.
+ 1. pause non-snapshot FATE ops (let running fate ops keep running, but pause
```

Comment:
could mention something about upgrades and how fate ops cannot be run in upgraded system

Reply:
I added a section about upgrades.  While writing it I thought of some issues with upgrades and possible solutions.